proto/jumpstarter/v1/jumpstarter.proto

```diff
@@ -170,16 +169,16 @@ message GetLeaseRequest {
 }
 
 message GetLeaseResponse {
-  google.protobuf.Timestamp begin_time = 1;
-  google.protobuf.Timestamp end_time = 2;
-  LabelSelector selector = 3;
-  optional string exporter_uuid = 4;
+  google.protobuf.Duration duration = 1;
+  LabelSelector selector = 2;
+  optional google.protobuf.Timestamp begin_time = 3;
+  optional google.protobuf.Timestamp end_time = 4;
+  optional string exporter_uuid = 5;
```

Comment:
Don't we get a lease name/uuid or something to keep polling later if we have to wait?